chainer/links/model/classifier.py

```diff
@@ -35,23 +36,32 @@ def __init__(self, predictor,
         self.loss = None
         self.accuracy = None
 
-    def __call__(self, x, t):
+    def __call__(self, *args):
         """Computes the loss value for an input and label pair.
 
         It also computes accuracy and stores it to the attribute.
 
         Args:
-            x (~chainer.Variable): Input minibatch.
-            t (~chainer.Variable): Corresponding ground truth labels.
+            args (list of ~chainer.Variable): Input minibatch.
+
+        The all elements of ``args`` but last one are features and
+        the last element corresponds to ground truth labels.
+        It feeds features to the predictor and compare the result
+        with ground truth labels.
 
         Returns:
             ~chainer.Variable: Loss value.
 
         """
+
+        if chainer.is_debug():
```

Comment:
I feel this condition is redundant because this assertion is enough small to execute.